readme.md

```diff
@@ -230,6 +263,6 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+var andy = Instructor.findOne({name: 'Andy'});
+andy.wishlist_items.push({'description: "Resin Laying Deer Figurine, Gold"'});
```

Comment:
Keep in mind that this is asynchronous, and that you'll need to `save` Andy after pushing the new data to him.